drivers: video: Fix blocking `video_buffer_alloc` (#80665)
zephyrproject-rtos/zephyr: doc/releases/migration-guide-4.1.rst

Device Drivers and Devicetree
*****************************

> * The :c:func:`video_buffer_alloc` and :c:func:`video_buffer_aligned_alloc` function in the
>   video API now includes a timeout parameter.

Reviewer: ```suggestion
* The :c:func:`video_buffer_alloc` and :c:func:`video_buffer_aligned_alloc` functions in the
  video API now take an additional timeout parameter.
```
